src/components/layoutStyles.css

```
@@ -0,0 +1,42 @@
+/* global.css */
+
+body {
+  --bg: #fff;
+  /* OC Gray 9 */
+  --textNormal: #212529;
+  --textTitle: #212529;
+  /* OC Blue 6 */
+  --textLink: #228be6;
+  --hr: hsla(0, 0%, 0%, 0.2);
+
+  background-color: var(--bg);
+}
+
+body.dark {
+  -webkit-font-smoothing: antialiased;
+
+  --bg: #343a40;
```

```suggestion
  --bg: #212529;
```

Let's make it darker Grey 9

---

I think the macOS default background color is `#1e1e1e` for dark mode.